docs/arch/adr-59.md

```diff
@@ -0,0 +1,18 @@
+# ADR 58: Incremental Static Regeneration for project pages
+
+## Status
+Accepted
+
+## Context
+When the first Engaging Crowds projects launched, project pages were built dynamically with `getServerSideProps`. Page builds were slow, with pages sometimes taking several seconds to load in the browser ([#2741](https://github.com/zooniverse/front-end-monorepo/issues/2741).)
+
+## Decision
+Use [Incremental Static Regeneration](https://nextjs.org/docs/pages/building-your-application/data-fetching/incremental-static-regeneration) (ISR) to build project pages. ISR is a feature of Next.js, which allows page HTML and JSON to be built and cached on demand, then served with `stale-while-revalidate` cache contol headers. While a page is building, visitors will get stale, cached content, resulting in a faster experience.
```

Comment:
Serving static content from the CDN is also financially cheaper for Zooniverse, compared with serving HTML and JSON responses from the Next.js origin server.